util/MultiplayerCommon.h

```diff
@@ -51,6 +51,13 @@ struct FO_COMMON_API GalaxySetupData {
     /** \name Mutators */ //@{
     void                SetSeed(const std::string& seed);
     void                SetGameUID(const std::string& game_uid);
+
+    /** HACK! This must be set to the encoding empire's id when serializing a
+      * Universe, so that only the relevant parts of the Universe are
+      * serialized.  The use of this global variable is done just so I don't
+      * have to rewrite any custom boost::serialization classes that implement
+      * empire-dependent visibility. */
+    int& EncodingEmpire();
```

Comment:
This is a struct with public members, so there's no point to this getter... just access `.m_encoding_empire` directly.

Reply:
Fixed.